samples/nextjs/scripts/generate-config.ts

```diff
@@ -35,8 +35,8 @@ export function generateConfig(configOverrides?: { [key: string]: string }): voi
   // The GraphQL endpoint is an example of making a _computed_ config setting
   // based on other config settings.
   const computedConfig: { [key: string]: string } = {};
-  computedConfig.graphqlEndpoint =
-    '`${config.sitecoreApiHost}${config.graphqlEndpointPath}?sc_apikey=${config.sitecoreApiKey}`';
+  computedConfig.graphQLEndpoint =
+    '`${config.sitecoreApiHost}${config.graphQLEndpointPath}?sc_apikey=${config.sitecoreApiKey}`';
```

Comment:
Probably out of scope for this ticket, but I think we should capture Sitecore's query string param names into constants.

Reply:
Agree, would be a good future enhancement (looks like all the frameworks do this) but out of scope here.